_posts/en/newsletters/2024-11-15-newsletter.md

```diff
@@ -0,0 +1,120 @@
+
+## News
+
+- **MAD-based offchain payment resolution (OPR) protocol:** John Law
+  [posted][law opr] to Delving Bitcoin the description of a micropayment
+  protocol that requires both participants to contribute funds to a bond
+  that can be effectively destroyed at any time by either participant.
+  This creates an incentive for both parties to appease the other or
+  risk mutually assured destruction (MAD) of the bonded funds.
+
+  This differs from the ideal of a _trustless protocol_ where only the
+  party at fault loses funds in the case of a protocol breach.  However,
+  in practice, deployed trustless protocols like LN often requires the
+  conforming party to pay onchain transaction fees to recover their
+  funds from a breach.  Law uses this fact to argue for some of the
+  benefits of a MAD-based protocol:
+
+  - In the case of destruction of funds, it uses much less blockchain
+    space than trustless contract enforcement, improving scalability.
+
+  - Being based on counterparty appeasement rather than global
+    consensus, the MAD-based protocol can enforce expiries as short as a
+    fraction of a second rather than a minimum of at least several
+    blocks.  Law gives the example of guaranteed payment resolution
+    (success or failure) in less than ten seconds compared to LN
+    currently needing up to two weeks in the worst case to settle a
+    payment.
+
+  - In the case of a prolonged communication failure between two
+    counterparties, the MAD-based protocol doesn't require either party
+    to put any data onchain (and both parties are incentivized not to do
+    so, as they will lose their portion of the bond deposit).  In a
+    protocol like [LN-Penalty][topic ln-penalty], pending [HTLCs][topic
+    htlc] in a channel must be settled onchain by a deadline if
+    communication breaks down.
+
+    Law emphasizes that this can make OPR much more efficient inside a
+    [channel factory][topic channel factories], [timeout tree][topic
+    timeout trees], or other nested structure that would ideally keep
+    the nested portions offchain.
+
+  Matt Morehouse [replied][morehouse opr] that appeasement can logically
+  lead to slow theft.  For example, Mallory claims Bob failed at an
+  operation that's worth 5% of the bond; Bob isn't sure he failed, but
+  paying Mallory 5% is better than losing his 50% of the bond, so he
+  acquiesces; Mallory repeats.  This problem is made worse by the
+  inability to prove fault in typical communication networks: if Mallory
+  and Bob lose contact for long enough for a failure to occur, they may
+  each blame the other, resulting in MAD.  Morehouse additionally notes
+  that OPR requires reserving more user funds for the bond deposits,
+  which may degrade UX---users today are already confused about the
+  [BOLT2][] _channel reserve_ that prevents them from spending more than
+  99% of channel balance.
+
+  Discussion was ongoing at the time of writing.
```

Comment:
```suggestion
  Discussion is ongoing at the time of writing.
```
Or maybe even, "Discussion is ongoing" (since it's assumed that this applies at the time of writing, when else could it be?)

Reply:
Prefer to keep as is. Searching through past newsletters, "discussion was ongoing" has been used 7 times and "discussion is ongoing" 0 times.

Reply:
Leaving as-is.  I think either way is correct and I find the past tense version more natural.